docs/how-to-run-simulated.adoc

```diff
@@ -157,7 +157,7 @@ Note that as the output indicates, this cluster will be available to anybody tha
 +
 [source,text]
 ----
-$ cargo run --bin omicron-dev -- ch-run
+$ cargo run --bin ch-dev run
```

Comment:
I noticed you dropped a bunch of the `--` arguments here.  Was that intentional?

Reply:
Ah honestly it doesn't matter, as long as the first argument doesn't start with double-dashes. But it is confusing.

Based on discussion in #oxide-control-plane I added `xtask` aliases which make this problem moot. (Note that it's still possible to run the commands separately, and the big concern we had with a unified binary behind xtask was that omicron-dev would have different sets of arguments based on the last xtask command run. That's not a concern any more.)